CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
```

Comment:
Is there a reason to bump the CMake version?

Reply:
required to be able to set the permissions with `configure_file`